src/System.Numerics.Vectors/tests/Performance/HashCodeTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xunit.Performance;
+using Xunit.Sdk;
+
+namespace System.Numerics.Performance.Tests
+{
+    public class HashCodeTest
+    {
+        private static int BaseHashCodeCombine(int h1, int h2)
+        {
+            return ((h1 << 5) + h1) ^ h2;
+        }
+
+        [Benchmark(InnerIterationCount = 1000)]
+        public void BaseHashTest()
```

Comment:
I agree. I run tests before/after modifications, but I needed somehow to commit performance tests. Extracted logic into separate method.